MapboxNavigationUI/MGLMapView.swift

```diff
@@ -9,8 +9,8 @@ let routeLayerCasingIdentifier = "routeLayerCasing"
 
 extension MGLMapView {
     
-    public func annotate(_ routes: [Route], clearMap: Bool) {
-        guard let route = routes.first, var coordinates = route.coordinates else {
+    public func annotate(_ route: Route) {
```

Comment:
I think this method originally took an array in anticipation of alternative route support, but the next line brushed that idea away anyways. 😄